drivers/SmartThings/sonos/src/api/cmd_handlers.lua

```diff
@@ -9,6 +9,8 @@ local st_utils = require "st.utils"
 --- https://developer.sonos.com/reference/control-api-examples-lan/
 local CapCommandHandlers = {}
 
+local QUEUE_ACTION_PREF = "queueAction"
```

Comment:
@TheMegamind The current preference value is stored on the device under the `preferences` table. The `name` defined in the .yml is the key. **The Lua value** will be **The YAML Key**. The YAML Value is used as the Display Name for the preference in the mobile phone app.

Reply:
@dljsjr Interesting. Different but less complicated than I expected.